### ISSUE TITLE
question: IProp origin attribute in case of chaining from action in the statement - anchors to action id not the instance of action id

### ISSUE THREAD

**Issue by tomaham** [DISSINET/InkVisitor#790]
See https://dissinet.cz/apps/inkvisitor-sandbox/#territory=4bc64aef-349c-4d97-b0e2-4b4eea315d3f&actant=1772800d-4cb3-45e7-bd20-0665333bb3e5&statement=609593f9-22a8-4404-aab4-509c5219ba8b .

![image](https://user-images.githubusercontent.com/3739529/154741185-66b538e0-4e54-48d7-97f0-279aabc92737.png)
![image](https://user-images.githubusercontent.com/3739529/154741214-86297c56-881d-4827-bbed-d65812bea2a2.png)

I would expect that that third IProp instance will originate in the **instance** of action A0221 (i.e. id 7b5c94ba-803a-4af3-8574-052387efea7d), not in the action A0221 itself.

**Comment by adammertel** [comment]
Yes, this should be happening in the new data model. Look at the `dev` branch, it should be working (still not 100% finished) there.

**Comment by tomaham** [comment]
This is not relevant anymore.
